packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -117,6 +117,12 @@ export class SfdxWorkspaceChecker implements PreconditionChecker {
   }
 }
 
+export class EmptyPreChecker implements PreconditionChecker {
```

Comment:
Command does not require any preconditions to be ran (unlike the other commands which all needed an sfdx workspace).